.github/workflows/deploy-to-control-plane.yml

```
@@ -34,79 +46,107 @@ jobs:
-        env:
-          CPLN_TOKEN: ${{ secrets.CPLN_TOKEN }}
         run: |
           if ! cpflow exists -a ${{ env.APP_NAME }}; then
             echo "No review app exists for this PR"
             exit 0
           fi
           echo "app_exists=true" >> $GITHUB_OUTPUT
 
+      - name: Validate Deployment Request
+        id: validate
+        run: |
```

actionlint (1.7.4)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)